e2e/scripts/st_map.py

```diff
@@ -22,7 +23,10 @@
 
 # Simple map.
 
-coords = np.random.randn(1000, 2) / [50, 50] + [37.76, -122.4]
+coords: "np.typing.NDArray[np.float_]" = cast(
+    Any,
+    np.random.randn(1000, 2) / [50, 50],
+) + [37.76, -122.4]
```

Comment:
nit: would probably be useful to leave a comment explaining why the cast here is needed (same comment in `st_pydeck_chart.py`)

Reply:
Added an explanation here as well. Hopefully, in time, most of these casts will be redundant.